webcompat/static/js/lib/issue-wizard-bugform.js

```diff
@@ -1373,7 +1373,12 @@ BugForm.prototype.getDataURIFromPreviewEl = function() {
 */
 BugForm.prototype.addImageURL = function(response) {
   var img_url = response.url;
-  var imageURL = ["[Screenshot](", img_url, ")"].join("");
+  var imageURL = [
+    "<details><summary>View the screenshot</summary>",
+    "<img alt='Screenshot' src='",
```

Comment:
typo?

` "<img alt='Screenshot' src='",` -> ` "<img alt='Screenshot' src=",`

Reply:
maybe is better using template string, more readable, what do you think?
`<details><summary>View the screenshot</summary><img alt='Screenshot' src="${img_url}"></details>`

Reply:
ah no the typo is elsewhere 😝 

```diff
-       "></details>"
+       "'></details>"
```

Reply:
template string seems to be a good idea.

Reply:
The one reason to not use a template strings (yet) would be cross-browser support: https://caniuse.com/#feat=template-literals.

Reply:
@miketaylr ah, yes.. true. even on webcompat? I mean about the stats of browsers version of webcompat user? But yes, maybe not a good idea after all.

Reply:
ok modification done. fixed the quote issue in the 3 instances where it appeared.

Reply:
> even on webcompat? I mean about the stats of browsers version of webcompat user?

I just checked, and IE users are 1.87% of our users (wow!). Not super high, but if we can avoid an error for them until they disappear it seems good.